lib/active_model/serializer.rb

```diff
@@ -42,9 +43,11 @@ def self.serializer_for(resource, options = {})
       end
     end
 
-    # @see ActiveModel::Serializer::Adapter.lookup
+    # @see ActiveModelSerializers::Adapter.lookup
+    # Deprecated
     def self.adapter
-      ActiveModel::Serializer::Adapter.lookup(config.adapter)
+      warn 'Calling adapter method in Serializer, please use the ActiveModelSerializers::configured_adapter'
+      ActiveModelSerializers::Adapter.lookup(config.adapter)
```

Comment:
Since this was deprecated, I just want to point that the tests for this were also removed. I reckon that the code here and in `ActiveModelSerializers::configured_adapter` are exactly the same and that the latter is not tested but I wanted to point it out anyway.